twitter/urls.py

```diff
@@ -14,10 +14,13 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from user.views import *
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', check_login, name='index'),
-]
+    path('', home, name='home'),
+    path('user/', include('user.urls')),
+]+ static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Should this `static` not in next line ?

Reply:
we can do in next line like if you say 
urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)

![image](https://github.com/GaganDureja/twitter/assets/53633318/d6f5579a-5e04-4eab-9633-ede2fe1f8855)